README.md

```diff
@@ -1,2 +1,30 @@
 # Galite
 Galite Framework
+
+![License](http://img.shields.io/badge/license-LGPL_v2.1-lightgrey.svg?style=flat)
+
+Welcome to GALITE, the new framework of [kopiLeft](https://github.com/kopiLeft) with expressive, elegant syntax, based on Kotlin, DSL.
+
+Galite allows users to build business applications with forms, reports and charts using a simple, creative and specific syntax that will allow you to put all the verbosity aside and concentrate on the business logic with it’s simple domain specific language . 
+
+## PURPOSE
+Since development must be as creative and enjoyable as doing common tasks, GALITE provides all the needed tools for developers to take the pain out of designing and implementing applications.
+It is a framework basically, that help them create forms to enter their data, reports and charts for visualizing them, all to control their stocks and items.
```

Comment:
a framework that basically helps to create forms, reports and charts , enter their data and visualize it all to control .....